.drone/drone.yml

```diff
@@ -7,63 +7,75 @@ platform:
   arch: amd64
 
 steps:
-- name: image-tag
-  image: alpine/git:v2.30.2
-  commands:
-  - apk --update --no-cache add bash
-  - git fetch origin --tags
-  - echo $(./tools/image-tag)-amd64 > .tags
-
-- name: build-tempo-binaries
-  image: golang:1.17-alpine
-  commands:
-  - apk add make git
-  - COMPONENT=tempo GOARCH=amd64 make exe
-  - COMPONENT=tempo-vulture GOARCH=amd64 make exe
-  - COMPONENT=tempo-query GOARCH=amd64 make exe
-
-- name: build-tempo-image
-  image: plugins/docker
-  settings:
-    build_args:
-    - TARGETARCH=amd64
-    dockerfile: cmd/tempo/Dockerfile
-    password:
-      from_secret: docker_password
-    repo: grafana/tempo
-    username:
-      from_secret: docker_username
-
-- name: build-tempo-vulture-image
-  image: plugins/docker
-  settings:
-    build_args:
-    - TARGETARCH=amd64
-    dockerfile: cmd/tempo-vulture/Dockerfile
-    password:
-      from_secret: docker_password
-    repo: grafana/tempo-vulture
-    username:
-      from_secret: docker_username
-
-- name: build-tempo-query-image
-  image: plugins/docker
-  settings:
-    build_args:
-    - TARGETARCH=amd64
-    dockerfile: cmd/tempo-query/Dockerfile
-    password:
-      from_secret: docker_password
-    repo: grafana/tempo-query
-    username:
-      from_secret: docker_username
+  - name: image-tag
+    image: alpine/git:v2.30.2
+    commands:
+      - apk --update --no-cache add bash
+      - git fetch origin --tags
+      - echo $(./tools/image-tag)-amd64 > .tags
+
+  - name: build-tempo-binaries
+    image: golang:1.17-alpine
+    commands:
+      - apk add make git
+      - COMPONENT=tempo GOARCH=amd64 make exe
+      - COMPONENT=tempo-vulture GOARCH=amd64 make exe
+      - COMPONENT=tempo-query GOARCH=amd64 make exe
+
+  - name: build-tempo-image
+    image: plugins/docker
+    settings:
+      build_args:
+        - TARGETARCH=amd64
+      dockerfile: cmd/tempo/Dockerfile
+      password:
+        from_secret: docker_password
+      repo: grafana/tempo
+      username:
+        from_secret: docker_username
+
+  - name: build-tempo-vulture-image
+    image: plugins/docker
+    settings:
+      build_args:
+        - TARGETARCH=amd64
+      dockerfile: cmd/tempo-vulture/Dockerfile
+      password:
+        from_secret: docker_password
+      repo: grafana/tempo-vulture
+      username:
+        from_secret: docker_username
+
+  - name: build-tempo-query-image
+    image: plugins/docker
+    settings:
+      build_args:
+        - TARGETARCH=amd64
+      dockerfile: cmd/tempo-query/Dockerfile
+      password:
+        from_secret: docker_password
+      repo: grafana/tempo-query
+      username:
+        from_secret: docker_username
+
+  - name: build-tempo-serverless-lambda
```

Comment:
Here lies the addition for `amd64`.  Perhaps we want `arm` also, but given that we're cloud, not sure it is needed.